snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -82,23 +82,23 @@ init_config:
 #        - TCP
 
 instances:
-  
+
   ## @param ip_address - string - required
   ## For SNMP v1 & v2 enter localhost
-  ## For SNMP v3 enter 192.168.34.10 
+  ## For SNMP v3 enter 192.168.34.10
```

Comment:
this actually doesn't make sense 🤦‍♂️ 
Let's just say `IP address of the device to monitor`

Reply:
```suggestion
  ## For SNMP v3 enter the IP address of the device to monitor.
```